doc-content/enterprise-only/openshift/hacep-client-proc.adoc

```diff
@@ -22,17 +22,7 @@ oc extract secret/my-cluster-cluster-ca-cert --keys=ca.crt --to=- > src/main/res
 keytool -import -trustcacerts -alias root -file src/main/resources/ca.crt -keystore src/main/resources/keystore.jks -storepass password -noprompt
 ----
 +
-. In the `src/main/resources` subdirectory of the project, create a `configuration.properties` file with the following content:
-+
-----
-ssl.keystore.location=keystore.jks
-ssl.truststore.location=keystore.jks
-ssl.keystore.password=password
-ssl.truststore.password=password
-bootstrap.servers=http://<serveraddress>
-----
-+
-Replace `<serveraddress>` with the address that the route for the Kafka server uses.
+. In the `src/main/resources` subdirectory of the project, edit the `configuration.properties` file. Replace `<bootstrap-hostname>`  with the address that the route for the Kafka server uses.
```

Comment:
This is two sentences so two steps. Consider something like this:
..of the project, open  the `configuration.properties` file and replace...

Also, is route a thing that uses an address? If not. try to rephrase 'that the route... uses...